docs/design.md

```diff
@@ -0,0 +1,65 @@
+# Designing ORAS
+
+## Introduction
+
+[ORAS](https://oras.land/) is a tool for working with OCI artifacts, especially ones in the OCI registries. It provides user-friendly interfaces to push and pull artifacts to / from registries as well as options for advanced scenarios.
+
+## Unified Experience
+
+The objective of ORAS is simple as transferring artifacts from one place to another.
+
+In the conventional [client-server model](https://en.wikipedia.org/wiki/Client%E2%80%93server_model), the operation of downloading artifacts from the remote registries is referred to as **pull**, and the operation of uploading artifacts from the remote registry is referred to as **push**.

```

Comment:
uploading artifacts "to"

Reply:
Thanks @mnltejaswini for pointing out the typo.

BTW, GitHub has a very nice feature named "insert a suggestion <ctrl+g>". Try it next time.

![image](https://user-images.githubusercontent.com/32161882/140475870-2bea1f26-3675-424d-9aa7-5b25364ad7dd.png)